README.md

```diff
@@ -1,2 +1,118 @@
+         # `file` is relative to .strigo directory
+         - file: "../Exercise.md"
+           title: "Title!"
+   ```
+
+1. In Strigo Class editor, fetch Exercises for a desired Class
+1. Done!
+
+---
+
+## Config file structure
+
+- `classes` - array of Classes
+  - `class_id` - defines the Class ID, optional if the configuration has only one Class
+  - `exercises` - array of Class' Exercises, cannot be empty
+    - `file` - path to Exercise markup file, relative to `.strigo` directory, supported extensions `.md` (markdown) and `.rst` (restructured), any other extension should be overriden by `syntax` field (see below)
+    - `title` - defines Exercise title in Strigo interface
+    - `syntax` - overrides `file` extension, supported syntaxes `.md` (markdown) and `.rst` (restructured)
+
+---
+
+## Main concepts
+
+1. **Hello world**
+
+   Just configure the order of the files, set titles and it is good to go.
+
+   ```yaml
+   ---
+   classes:
+     - exercises:
+         # `file` is relative to .strigo directory
+         - file: "../hello-world/hello.md"
+           title: "Hello!"
+         - file: "../hello-world/world.txt"
+           title: "World!"
+   ```
+
+1. **Multiple Classes in a single repository**
+
+   You can have multiple Classes in a single repository, defining Class IDs for each.
+
+   ```yaml
+   ---
+   classes:
+     - class_id: "<class_id_1>"
+       exercises:
+         - file: "../1.md"
+           title: "Title 1!"
+         - file: "../2.md"
+           title: "Title 2!"
+     - class_id: "<class_id_2>"
+       exercises:
+         - file: "../3.md"
+           title: "Title 3!"
+         - file: "../4.md"
+           title: "Title 4!"
+   ```
+
+1. **Reusing Exercises for different classes**
+
+   There is no restriction on using the same file for multiple Classes, or even using the same file multiple times for the same Class
+
+   ```yaml
+   ---
+   classes:
+     - class_id: "<class_id_1>"
+       exercises:
+         - file: "../1.md"
+           title: "Title 1!"
+         - file: "../1.md"
+           title: "Title 1!"
+     - class_id: "<class_id_2>"
+       exercises:
+         - file: "../1.md"
+           title: "Title 1!"
+   ```
+
+1. **Markup syntaxes**
+
+   You can use different markup syntaxes by using `syntax` field for each Exercise
+
+   ```yaml
+   ---
+   classes:
+     - class_id: "<class_id_1>"
+       exercises:
+         - file: "../1.txt"
+           title: "Markdown!"
+           # `syntax` is optional
+           # can be "md" | "rst"
+           # uses file extension as fallback
+           syntax: "md"
+         - file: "../2.txt"
+           title: "Title 1!"
+           syntax: "rst"
+   ```
+
+1. **Relative vs Absolute paths to images**
+
+   Your Exercises' repository can have images stored inside the repository, a.k.a Relative images. Fetching these exercises will upload these images to Strigo. If the image could not be accessed, we will replace it with a placeholder instead.
```

Comment:
"Your exercises repository can have images stored within it. Fetching these exercises will upload the images to Strigo. If the image could be accessed for some reason, we will replace it with a placeholder instead"

"Absolute image paths will not be affected, so if the image cannot be accessed, it won't show up for your students."